gradebookng/impl/src/main/webapp/WEB-INF/components.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd">
+
+    <bean id="org.sakaiproject.util.ResourceLoader.grading"
+          class="org.sakaiproject.util.ResourceLoader">
+    </bean>
+
+	<bean id="gradingHibernateMappings"
```

Comment:
bean id should be org.sakaiproject.springframework.orm.hibernate.impl.AdditionalHibernateMappingsImpl.gradebook